README.md

```diff
@@ -145,7 +166,7 @@ code, include a script or makefile that will do the job.
 
 ## A2.5
 
-«replace this with your answer»
```

Comment:
# -12

I know you said that you didn't have time to test this, but there are two fundamental problems with the code that's here:

1. It doesn't compile

2. It doesn't use a table to implement the FSM

Sorry